radixdlt-java/src/main/java/com/radixdlt/client/application/RadixApplicationAPI.java

```diff
@@ -324,7 +338,7 @@ public static RadixApplicationAPI create(RadixIdentity identity) {
 	 * @return Disposable to dispose to stop pulling
 	 */
 	public Disposable pull() {
-		return pull(getMyAddress());
+		return this.pull(this.getMyAddress());
```

Comment:
These changes are really unhelpful.  Is this another save action thing that needs to be turned off?

Reply:
As said in another comment, I've disabled everything there is to disable in terms of reformatting and auto-actions. I hope this fixes this issue. Sorry about that.